src/subscript/field_statistics/field_statistics.py

```diff
@@ -1042,6 +1064,13 @@ def get_specifications(input_dict, ertbox_size, ert_config_path):
         use_geogrid_fields = True
         geogrid_fields_dict = input_dict["geogrid_fields"]
 
+        key = "geogrid_name"
+        if key in geogrid_fields_dict:
+            geogrid_name = geogrid_fields_dict[key]
+            geogrid_name = geogrid_name.strip()
+        else:
+            raise KeyError(f"Missing keyword:  {key} specifying name of geomodel grid.")
+
```

Comment:
I propose to have this optional to keep compatibility with current user setup. New user will have possibility to specify `geogrid_name`, but current user can still use the default value `geogrid`.

```suggestion
        geogrid_name = geogrid_fileds_dict.get("geogrid_name", "geogrid").strip()
```

Reply:
OK Fixed